torch/nn/quantized/functional.py

```diff
@@ -118,6 +118,63 @@ def conv2d(input, weight, bias,
                                       stride, padding, dilation,
                                       groups, scale, zero_point)
 
+def conv3d(input, weight, bias, stride=1, padding=0, dilation=1, groups=1,
+           padding_mode='zeros', scale=1.0, zero_point=0, dtype=torch.quint8):
+    r"""
+    Applies a 3D convolution over a quantized 3D input composed of several input
+    planes.
+
+    See :class:`~torch.nn.quantized.Conv3d` for details and output shape.
+
+    Args:
+        input: quantized input tensor of shape
+          :math:`(\text{minibatch} , \text{in\_channels} , iD , iH , iW)`
+        weight: quantized filters of shape
+          :math:`(\text{out\_channels} , \frac{\text{in\_channels}}{\text{groups}} , kD , kH , kW)`
```

Comment:
Want to check: is it N, C, H, W, D, or N, C, D, H, W?

Reply:
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/qconv.cpp#L284 Here it is H, W, D. But I also see https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/qconv.cpp#L124 Here is D, H, W.

Reply:
That is NCDHW, the first two dims are N and C, so D is at dim(2).